### ISSUE TITLE
[ADDITION] Bleachbit (Complete)

### ISSUE THREAD

**Issue by Merely-Human** [Lissy93/awesome-privacy#302]
### Category

Security Tools

### Listing Name

Bleachbit

### Listing URL

https://www.bleachbit.org/

### Listing Icon

https://www.bleachbit.org/sites/default/files/zen_classic_logo_0.png

### Listing Description

When your computer is getting full, BleachBit quickly frees disk space. When your information is only your business, BleachBit guards your privacy. With BleachBit you can free cache, delete cookies, clear Internet history, shred temporary files, delete logs, and discard junk you didn't know was there. Designed for Linux and Windows systems, it wipes clean thousands of applications including Firefox, Adobe Flash, Google Chrome, Opera, and more. Beyond simply deleting files, BleachBit includes advanced features such as shredding files to prevent recovery, wiping free disk space to hide traces of files deleted by other applications, and vacuuming Firefox to make it faster. Better than free, BleachBit is open source.

### GitHub repository

https://github.com/bleachbit

### Is Open Source?

- [X] Yes, it's open source

### Security Audited?

- [X] Yes, it's been security audited

### Accepts Anon Payment?

- [X] Yes, accept anonymous payment

### Justification and Further Info

> Features
> Concerned about privacy and disk space? BleachBit is your solution! In today's digital world, it’s essential to protect your personal information. With just a few clicks, you can easily delete unwanted files and permanently erase sensitive data, ensuring your confidentiality and giving you peace of mind.
> 
> BleachBit has many useful features designed to help you easily your computer to free space and maintain privacy.
> 
> Shred sensitive files, so one can ever read their contents.
> You always have the "Pro" version: BleachBit is free of charge
> Anonymous: you do not need to register an account
> Simple to operate: read the descriptions, check the boxes you want, click preview, and click delete.
> Multi-platform: it runs on both Linux and Windows
> What's hiding in your software? BleachBit is free to share, learn, and modify because it is open source, so there are no secrets.
> No adware, spyware, telemetry, malware, bloatware, backdoors, browser toolbars, or "value-added software"
> Translated to 66 languages
> Overwrite free disk space to hide previously deleted files
> On Windows, choose to either install or run in portable mode. like from a USB drive.
> Command line interface for scripting and automation
> Advanced users can write their own cleaners using CleanerML
> Automatically import and update winapp2.ini cleaner files giving Windows users access to thousands of additional cleaners
> Uses
> Use BleachBit to:
> 
> Free disk space
> Reduce the size of backups and the time to create them by removing unnecessary files
> Maintain privacy
> Improve system performance
> Prepare system for backups by removing junk data that wastes space and time.
> Prepare whole disk images for compression (common for "ghost" backups and virtual machines) by wiping free disk space
> Cleaners
> Cleaners contain the rules for cleaning specific applications, and BleachBit includes a growing list of cleaners for Google Chrome, Edge, Firefox, VLC, system logs, temporary files, and many others.
> 
> Many more cleaners for Windows and Linux are available in the CleanerML community repository.
> 
> Add thousands of cleaners by using winapp2.ini (Windows only).
> 
> Advanced cleaners
> Going beyond standard deletion of files, BleachBit has several advanced cleaners:
> 
> Clear the memory and swap on Linux
> Delete broken shortcuts on Linux
> Delete the Firefox URL history without deleting the whole file—with optional shredding
> Delete Linux localizations: delete languages you don't use. More powerful than localepurge and available on more Linux distributions.
> Clean APT for Debian, Ubuntu, Kubuntu, Xubuntu, and Linux Mint
> Find widely-scattered junk such as Thumbs.db and .DS_Store files.
> Execute yum clean for CentOS, Fedora, and Red Hat to remove cached package data
> Delete Windows registry keys—often where MRU (most recently used) lists are stored
> Delete the OpenOffice.org recent documents list without deleting the whole Common.xcu file
> Overwrite free disk space to hide previously files
> Vacuum Firefox, Google Chrome, Liferea, Thunderbird, and Yum databases: shrink files without removing data to save space and improve speed
> Surgically remove private information from .ini and JSON configuration files and SQLite3 databases without deleting the whole file
> Overwrite data in SQLite3 before deleting it to prevent recovery (optional)
> Screenshots
> See screenshots.
> 
> Video
> Watch the videos.

**YAML Content for Addition**

```yaml
- name: Bleachbit
  url: https://www.bleachbit.org/
  icon: https://www.bleachbit.org/sites/default/files/zen_classic_logo_0.png
  description: >-
    When your computer is getting full, BleachBit quickly frees disk space. When
    your information is only your business, BleachBit guards your privacy. With
    BleachBit you can free cache, delete cookies, clear Internet history, shred
    temporary files, delete logs, and discard junk you didn't know was there.
    Designed for Linux and Windows systems, it wipes clean thousands of
    applications including Firefox, Adobe Flash, Google Chrome, Opera, and more.
    Beyond simply deleting files, BleachBit includes advanced features such as
    shredding files to prevent recovery, wiping free disk space to hide traces
    of files deleted by other applications, and vacuuming Firefox to make it
    faster. Better than free, BleachBit is open source.
  github: https://github.com/bleachbit
  openSource: true
  securityAudited: true
  acceptsCrypto: true

```

<sup>This ticket was submitted via <a href="https://awesome-privacy.xyz/submit">awesome-privacy.xyz/submit</a></sup>

### Final Checklist

- [X] I confirm that I've read and followed the [Contributing Guide](https://github.com/Lissy93/awesome-privacy/blob/main/.github/CONTRIBUTING.md)
- [X] I agree to follow the [Code of Conduct](https://github.com/Lissy93/awesome-privacy/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I have clearly declared any affiliation I may have with this service

**Comment by Merely-Human** [comment]
BleachBit is indeed a security tool. Here’s a breakdown of its features and capabilities that support its classification as a security tool:

Data destruction: BleachBit uses industry-standard encryption algorithms to overwrite deleted files, making it difficult to retrieve previously stored data. This ensures the confidentiality of deleted data, a critical aspect of security.
Secure deletion: BleachBit’s advanced cleaners can overwrite free disk space, making it challenging to recover previously deleted files. This feature helps protect against data breaches and unauthorized access.
Privacy-focused: BleachBit is designed with privacy in mind, aiming to maintain user data confidentiality. Its open-source nature and transparency in code and development processes contribute to its reputation as a trustworthy security tool.
System cleaning: BleachBit removes junk data, temporary files, and system logs, which can help prevent security vulnerabilities and maintain system integrity.
Customizable cleaners: Users can write their own cleaners using CleanerML, an open standard XML-based markup language, allowing for tailored security configurations.
Command-line interface: BleachBit offers a command-line interface for scripting and automation, enabling advanced users to integrate it with other security tools and workflows.
Open-source and community-driven: BleachBit’s open-source nature and community involvement ensure that its code is regularly reviewed and updated, reducing the risk of security vulnerabilities and backdoors.
In conclusion, BleachBit’s features and capabilities demonstrate its role as a security tool, specifically designed to protect user data, maintain system integrity, and ensure confidentiality. Its open-source nature, customizable cleaners, and command-line interface further reinforce its position as a trusted security solution.

But thank you for your opinion not based on facts.
